content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,567 @@
+---
+title: Building a Connector
+spelling:
+  cSpell:ignore spanmetrics servicegraph exampleconnector struct Errorf
+  cSpell:ignore mapstructure pdata mapstructure pmetric ptrace uber gord
+  cSpell:ignore gomod loggingexporter batchprocessor otlpreceiver
+---
+
+## Connectors in OpenTelemetry
+
+The content of this page is most applicable if you already have an instrumented
+application generating some kind of tracing telemetry data and already have an
+understanding of the [OpenTelemetry Collector](/docs/collector).
+
+## What is a Connector?
+
+Each pipeline in the OpenTelemetry Collector acts on one type of telemetry data.
+There may exist the need to process one form of telemetry data into another one,
+but it is required to route the according data to its proper collector pipeline.
+A connector acts as the means of sending telemetry data between different
+collector pipelines by connecting them. A connector acts as an exporter to one
+pipeline and a receiver to another.
+
+## Why use a Connector?
+
+There are processors in OpenTelemetry that convert telemetry data of one type
+into another one. A few examples are the spanmetrics processor, as well as the
+servicegraph processor. The spanmetrics processor generates aggregate requests,
+error and duration metrics from span data. T​he servicegraph processor analyzes
+trace data and generates metrics that describe the relationship between the
+services. Both these processors ingest trace data and convert them to metrics
+data. Since pipelines in the OpenTelemetry collector are for only one type of
+data, it is necessary to convert the trace data from the processor in the traces
+pipeline and send it to the metrics pipeline. Historically, some processors
+transmitted data by making use of a work-around that follows a bad practice
+where a processor directly exports data after processing. The connector
+component solves the need for this work-around and the processors that used the
+work around have been deprecated.
+
+The connector is beneficial at merging, routing and replicating data streams.
+Along with sequential pipelining, which is to connect pipelines together, the
+connector component is capable of conditional data flow and generated data
+streams. Conditional data flow means sending data to the highest priority
+pipeline and has error detection to route to alternative pipeline if need be.
+Generated data streams means that the component generates and emits its own data
+based on the received data. Additional details about the connector's full
+capabilities can be found at the following links:
```

Comment:
Similarly, the first paragraph focuses directly on a subset of connectors. I think this may present too narrow an understanding of their purpose. The second paragraph is more generalized. Do you think it makes sense to lead with the broader explanation and then dive into the examples?